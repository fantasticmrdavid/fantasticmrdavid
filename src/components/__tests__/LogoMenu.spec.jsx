import React from 'react';
import { shallow } from 'enzyme';
import Ui from 'components/LogoMenu/ui';

const toggleActive = jest.fn();

describe('Component -> LogoMenu', () => {
  it('should render the LogoMenu UI without throwing an error', () => {
    expect(shallow(<Ui toggleActive={toggleActive} />)).toMatchSnapshot();
  });
});
