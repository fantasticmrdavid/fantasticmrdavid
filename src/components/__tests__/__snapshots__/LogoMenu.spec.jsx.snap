// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component -> LogoMenu should render the LogoMenu UI without throwing an error 1`] = `
<styles__Container
  isOpen={false}
  onClick={[Function]}
>
  <styles__Title>
    Fantastic Mr David
  </styles__Title>
  <styles__Nav
    isOpen={false}
  >
    <withRouter(undefined)
      current={false}
      key="MenuLink_about"
      label="About"
      menuActive={false}
      target="about"
    />
    <withRouter(undefined)
      current={false}
      key="MenuLink_work"
      label="Latest Work"
      menuActive={false}
      target="work"
    />
    <withRouter(undefined)
      current={false}
      key="MenuLink_contact"
      label="Contact"
      menuActive={false}
      target="contact"
    />
  </styles__Nav>
  <styles__Subtitle
    isOpen={false}
  >
    <h2>
      <styles__CogIcon
        icon={
          Object {
            "icon": Array [
              512,
              512,
              Array [],
              "f013",
              "M487.4 315.7l-42.6-24.6c4.3-23.2 4.3-47 0-70.2l42.6-24.6c4.9-2.8 7.1-8.6 5.5-14-11.1-35.6-30-67.8-54.7-94.6-3.8-4.1-10-5.1-14.8-2.3L380.8 110c-17.9-15.4-38.5-27.3-60.8-35.1V25.8c0-5.6-3.9-10.5-9.4-11.7-36.7-8.2-74.3-7.8-109.2 0-5.5 1.2-9.4 6.1-9.4 11.7V75c-22.2 7.9-42.8 19.8-60.8 35.1L88.7 85.5c-4.9-2.8-11-1.9-14.8 2.3-24.7 26.7-43.6 58.9-54.7 94.6-1.7 5.4.6 11.2 5.5 14L67.3 221c-4.3 23.2-4.3 47 0 70.2l-42.6 24.6c-4.9 2.8-7.1 8.6-5.5 14 11.1 35.6 30 67.8 54.7 94.6 3.8 4.1 10 5.1 14.8 2.3l42.6-24.6c17.9 15.4 38.5 27.3 60.8 35.1v49.2c0 5.6 3.9 10.5 9.4 11.7 36.7 8.2 74.3 7.8 109.2 0 5.5-1.2 9.4-6.1 9.4-11.7v-49.2c22.2-7.9 42.8-19.8 60.8-35.1l42.6 24.6c4.9 2.8 11 1.9 14.8-2.3 24.7-26.7 43.6-58.9 54.7-94.6 1.5-5.5-.7-11.3-5.6-14.1zM256 336c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z",
            ],
            "iconName": "cog",
            "prefix": "fas",
          }
        }
        isOpen={false}
      />
      Art + Engineering
    </h2>
    <Memo() />
  </styles__Subtitle>
  <styles__Cta
    isOpen={false}
  >
    <styles__Icon
      icon={
        Object {
          "icon": Array [
            320,
            512,
            Array [],
            "f107",
            "M143 352.3L7 216.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.2 9.4-24.4 9.4-33.8 0z",
          ],
          "iconName": "angle-down",
          "prefix": "fas",
        }
      }
    />
  </styles__Cta>
</styles__Container>
`;
